Patch by Robert Scheck <robert@fedoraproject.org> for ifstatus >= 1.1.0,
which adds a few missing #include lines to not violate the C++ standards
and to build successfully with G++ version 4.3 and 4.4.

--- ifstatus/IFStatus.cc		2006-01-17 00:18:46.000000000 +0100
+++ ifstatus/IFStatus.cc.gcc44		2009-04-14 00:42:04.000000000 +0200
@@ -22,6 +22,7 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#include <cstdlib>
 #include "Main.h"
 #include "IFStatus.h"
 #include "Graph.h"
--- ifstatus/GraphDecorator.cc		2006-01-17 01:00:59.000000000 +0100
+++ ifstatus/GraphDecorator.cc.gcc44	2009-04-14 00:41:38.000000000 +0200
@@ -21,7 +21,8 @@
  *  along with this program; if not, write to the Free Software
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
- 
+
+#include <cstdlib> 
 #include "GraphDecorator.h"
 #include "Config.h"
 
--- ifstatus/Interface.cc		2006-01-17 02:48:27.000000000 +0100
+++ ifstatus/Interface.cc.gcc44		2009-04-14 00:42:23.000000000 +0200
@@ -22,6 +22,7 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#include <cstdlib>
 #include "Interface.h"
 #include "Config.h"
 
--- ifstatus/Config.cc			2005-02-26 12:54:51.000000000 +0100
+++ ifstatus/Config.cc.gcc44		2009-04-14 00:40:13.000000000 +0200
@@ -22,6 +22,7 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#include <cstdlib>
 #include "Main.h"
 #include "Config.h"
 #include "Util.h"
--- ifstatus/Email.cc			2005-02-26 12:54:51.000000000 +0100
+++ ifstatus/Email.cc.gcc44		2009-04-14 00:40:44.000000000 +0200
@@ -22,6 +22,7 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#include <cstdlib>
 #include "Email.h"
 #include "Config.h"
 
--- ifstatus/Interfaces.cc		2006-01-16 23:45:33.000000000 +0100
+++ ifstatus/Interfaces.cc.gcc44	2009-04-14 00:42:45.000000000 +0200
@@ -22,6 +22,7 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#include <cstdlib>
 #include "Interfaces.h"
 #include "Config.h"
 #include "Util.h"
